Description: ext/pcre tests that print messages of the library: the wording of PCRE2 10.37+ ("atomic assertion expected", the offset after the character) as well as the old one, as php-src has them since 8.1 and 8.3.
Origin: php-src ext/pcre/tests of PHP-8.1 (offset %r(0|1)%r, %r(1|2)%r) and PHP-8.3 (bug75457.phpt)

--- a/ext/pcre/tests/bug75457.phpt
+++ b/ext/pcre/tests/bug75457.phpt
@@ -6,5 +6,5 @@
 var_dump(preg_match($pattern, "hello"));
 ?>
 --EXPECTF--
-Warning: preg_match(): Compilation failed: assertion expected after (?( or (?(?C) at offset 8 in %sbug75457.php on line %d
+Warning: preg_match(): Compilation failed:%r( atomic|)%r assertion expected after (?( or (?(?C) at offset 8 in %sbug75457.php on line %d
 bool(false)
--- a/ext/pcre/tests/grep2.phpt
+++ b/ext/pcre/tests/grep2.phpt
@@ -29,7 +29,7 @@
 Warning: preg_grep() expects parameter 2 to be array, int given in %sgrep2.php on line 4
 NULL
 
-Warning: preg_grep(): Compilation failed: quantifier does not follow a repeatable item at offset 0 in %sgrep2.php on line 5
+Warning: preg_grep(): Compilation failed: quantifier does not follow a repeatable item at offset %r(0|1)%r in %sgrep2.php on line 5
 bool(false)
 array(3) {
   [5]=>
--- a/ext/pcre/tests/pcre_extra.phpt
+++ b/ext/pcre/tests/pcre_extra.phpt
@@ -10,5 +10,5 @@
 --EXPECTF--
 int(1)
 
-Warning: preg_match(): Compilation failed: unrecognized character follows \ at offset 1 in %spcre_extra.php on line 4
+Warning: preg_match(): Compilation failed: unrecognized character follows \ at offset %r(1|2)%r in %spcre_extra.php on line 4
 bool(false)
--- a/ext/pcre/tests/split.phpt
+++ b/ext/pcre/tests/split.phpt
@@ -20,7 +20,7 @@
 Warning: preg_split() expects at least 2 parameters, 0 given in %ssplit.php on line 3
 bool(false)
 
-Warning: preg_split(): Compilation failed: quantifier does not follow a repeatable item at offset 0 in %ssplit.php on line 4
+Warning: preg_split(): Compilation failed: quantifier does not follow a repeatable item at offset %r(0|1)%r in %ssplit.php on line 4
 bool(false)
 array(3) {
   [0]=>
